modules/hdf5_handler/HDF5CFUtil.cc

```diff
@@ -1274,25 +1275,28 @@ void HDF5CFUtil::cha_co(string &co,const string & vpath) {
                 if(co_rp_sep_pos[0]==0) {
                     // Obtain the '../' position at co 
                     if(co_rp_sep_pos.size() <var_sep_pos.size()) {
-                    size_t var_prefix_pos=var_sep_pos[var_sep_pos.size()-co_rp_sep_pos.size()-1];
+                        size_t var_prefix_pos=var_sep_pos[var_sep_pos.size()-co_rp_sep_pos.size()-1];
+                        string var_prefix=vpath.substr(1,var_prefix_pos);
+                        string co_suffix = co.substr(co_rp_sep_pos[co_rp_sep_pos.size()-1]+rp_sep.size());
+                        co = var_prefix+co_suffix;
+#if 0
```

Comment:
I'm not sure what your plans are for the groups of lines with the #if 0 ... #endif brackets - I usually leave these in for one PR and then remove them to make it easier to read what the code is doing. 

Sometimes there are things we really want to keep longer. Anyway, if you know these are never going to be used, we could remove them now and make the software a little easier to understand.

Reply:
These #if 0 #endif better kept for a while. Most of them contains useful information. I will see if I can clean up some in the next round.